### PR TITLE
Addlayout

### PR DESCRIPTION
**Pull request by NguyenHuy3003**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +47,13 @@ Configure github action at https://docs.github.com/en/pages/getting-started-with
 3. push pages to github. A build action will be triggered automatically to release the page https://github.com/hippopenny/hippopenny.github.io/actions
 
 
+## add hippo
++ Thêm file json vào ./_data đinh dạng tương tự
++ Thêm file image.png vào ./assets/imgages/scores
++ Thêm file .md vào ./scores định dạng tương tự
++ Chú ý tên file, file .md đổi tên theo tên game vd: permalink: /meta-score/alan-wake-ii (tên game)
+
```
Comment (tomcauldron) [inline README.md:55]: try to write in English please

File: assets/images/scores/2023-12-26-animal-crossing-wild-world.png
Comment (tomcauldron) [inline assets/images/scores/2023-12-26-animal-crossing-wild-world.png:1]: can we replace this picture? It's not visual appealing --- it's not cropped correctly either.

File: scores/2023-12-28-animal-crossing-new-leaf.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: HippoPenny Score
+layout: scoredetail
```
Comment (tomcauldron) [inline scores/2023-12-28-animal-crossing-new-leaf.md:3]: can you please share a screenshot somewhere?

**Review by tomcauldron — APPROVED** [review]
LGTM. please upload all other reviewers before merging to main

File: README.md (hunk 1 above)
Comment (NguyenHuy3003) [reply README.md:55]: done

File: assets/images/scores/2023-12-26-animal-crossing-wild-world.png
Comment (NguyenHuy3003) [reply assets/images/scores/2023-12-26-animal-crossing-wild-world.png:1]: done

File: scores/2023-12-28-animal-crossing-new-leaf.md (hunk 2 above)
Comment (NguyenHuy3003) [reply scores/2023-12-28-animal-crossing-new-leaf.md:3]: done

**Pull request merged**